src/modules/ROOT/pages/changelog/202309/05.json

```diff
@@ -0,0 +1,110 @@
+{
+  "meta": {
+    "releasedOn": "2024.03.21"
+  },
+
+  "added": [
+    {
+      "what": [
+        "the option to access user information like username, handle, email, etc. in pages, charts, and record fields."
+      ],
+      "why": [
+        "to provide an easier way for the user to access their user data."
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza/pull/1735"
+      ]
+    },
+    {
+      "what": [
+        "support for User and Record fields to be exported/imported on the record list page block."
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza/commit/7fc4ca921"
+      ]
+    },
+    {
+      "what": [
+        "the option to open a link/route/record page in a new tab using {APP_NAME_WORKFLOW} prompts."
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza/pull/1726"
+      ]
+    }
+  ],
+
+  "changed": [
+    {
+      "what": [
+        "the flow of updating values in fields and page blocks to fetch the latest values after an update."
+      ],
+      "why": [
+        "to ensure fields and blocks are always up to date."
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza/pull/1634"
+      ]
+    }
+  ],
+
+  "fixed": [
+    {
+      "what": [
+        "not being able to upload `.png` file to an application in {APP_NAME_ADMIN}."
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza/commit/0456fe4f3e2e127dbccc76d131bbd3d706829133"
+      ]
+    },
+    {
+      "what": [
+        "not being able to make a copy of predefined CRM namespace."
```

Comment:
Don't think this was exclusive to the CRM namespace; think it was for all namespaces.
Best you double-check with Jože but I think that was the case